src/DOM/HTML/Indexed/LengthLiteral.purs

```diff
@@ -0,0 +1,12 @@
+module DOM.HTML.Indexed.LengthLiteral where
+
+import Prelude (show, (<>))
+
+data LengthLiteral
+  = Pixels Int
```

Comment:
Urgh, it's a mess. For `HTMLImageElement` it's `unsigned long`, for `HTMLIFrameElement` it's `DOMString`, etc.